internal/nginx/config/generator.go

```diff
@@ -54,11 +56,32 @@ func generate(host state.Host, serviceStore state.ServiceStore) (server, Warning
 			warnings.AddWarning(r.Source, err.Error())
 		}
 
-		loc := location{
-			Path:      g.Path,
-			ProxyPass: generateProxyPass(address),
+		match, exists := r.GetMatch()
+		if exists && matchLocationNeeded(match) {
+			// FIXME(osborn): route index is hardcoded to 0 for now.
```

Comment:
Would it make sense to use our GitHub ids here? This is to accommodate potential external contributors. I used mine for my FIXMEs, but here https://github.com/nginxinc/nginx-kubernetes-gateway/blob/1c650a7ac770b5a39eed46ac8aacefc6dad9a396/cmd/gateway/setup.go#L27 @f5yacobucci used  https://github.com/yacobucci  rather than https://github.com/f5yacobucci (not sure if it was intentional)

Reply:
Good callout. Let's collapse on GitHub ids. I'll change mine and the lone yacobucci.